iree/compiler/Dialect/HAL/Target/MetalSPIRV/SPIRVToMSL.cpp

```diff
@@ -19,7 +19,7 @@
 #include "llvm/ADT/STLExtras.h"
 #include "llvm/Support/Debug.h"
 #include "llvm/Support/raw_ostream.h"
-#include "third_party/spirv_cross/spirv_msl.hpp"
+#include "spirv_cross/spirv_msl.hpp"
```

Comment:
keep `third_party/`, remove `INCLUDES` in the CMake file

see https://github.com/google/iree/blob/8df7e5cd90d4874ec0cc20f8aa3d3926d937025d/build_tools/cmake/iree_copts.cmake#L70-L76

Reply:
got it